packages/android_intent/android/src/main/java/io/flutter/plugins/android_intent/AndroidIntentPlugin.java

```diff
@@ -0,0 +1,85 @@
+package io.flutter.plugins.android_intent;
```

Comment:
@tvolkert I saw Todd's CL; so only package designations (not the paths) need to be updated? Typically java package and path needs to match.

https://github.com/flutter/plugins/tree/master/packages/shared_preferences/android/src/main/java/io/flutter/plugins/shared_preferences